app/views/miq_request/_prov_field.html.haml

```diff
@@ -304,8 +304,7 @@
                                :bs_tree       => @all_tags_tree,
                                :oncheck       => "miqOnCheckProvTags",
                                :check_url     => "/miq_request/prov_field_changed/#{check}",
-                               :checkboxes    => true,
-                               :cookie_prefix => "prov_trees"})
```

Comment:
These 2 bits seem related:

```
app/controllers/vm_common.rb
1205:        presenter[:clear_tree_cookies] = "prov_trees"

app/views/miq_request/_prov_edit.html.haml
8:  miqDeleteTreeCookies('prov_trees')
```

Reply:
Okay, I'm replacing the `prov_trees` with `all_tags_tree` which is the actual name of the tree and it will be stored under this name in the local storage.